### PR TITLE
Collaboration blog post

### PR DESCRIPTION
**Pull request by garethx**
Adds collaboration blog post.

Waiting on the correct gif.

- [x] Documentation
- [x] Testing
- [x] Ready to be merged

### CODE REVIEW

**Comment by lbogdan** [comment]
Build for latest commit https://github.com/codesandbox/codesandbox-client/pull/3721/commits/3a29ca7bd7e71f34b4a5649236e650fe45504f82 failed.

File: packages/homepage/content/articles/redefining-code-collaboration/index.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+date: 2020-03-25
+---
+
+Collaborating on code can be hard. Historically, web development has been seen as a solo pursuit with tooling being built for individual developers writing code locally. In reality, we rarely code by ourselves. While we may often physically edit code alone, we create things collectively—working within teams and contributing to group projects. Now more than ever, we need to be able to collaborate on code. Current events have forced many to embrace remote work, but developers have been leading that charge for years. Flexible working arrangements and distributed or hybrid teams are now commonplace, forcing a fundamental change in how we all create together.
+
+[CodeSandbox Live](https://codesandbox.io/post/introducing-codesandbox-live-real-time-code-collaboration-in-the-browser) made collaboration easier by giving you the ability to code in real-time with others. It's used for pair programming, at hackathons, during job interviews, and in classrooms, enabling two or more folks to work on code simultaneously. What's more, our [basic team support](https://codesandbox.io/dashboard/teams/new) has reduced friction in sharing ownership of sandboxes within a group.
```
Comment (MichaelDeBoey) [inline packages/homepage/content/articles/redefining-code-collaboration/index.md]:
```suggestion
[CodeSandbox Live](/post/introducing-codesandbox-live-real-time-code-collaboration-in-the-browser) made collaboration easier by giving you the ability to code in real-time with others. It's used for pair programming, at hackathons, during job interviews, and in classrooms, enabling two or more folks to work on code simultaneously. What's more, our [basic team support](/dashboard/teams/new) has reduced friction in sharing ownership of sandboxes within a group.
```

File: packages/homepage/content/articles/redefining-code-collaboration/index.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+<div class="typeform-widget" data-url="https://codesandbox.typeform.com/to/ktD2rd" style="width: 100%; height: 350px;"></div> <script> (function() { var qs,js,q,s,d=document, gi=d.getElementById, ce=d.createElement, gt=d.getElementsByTagName, id="typef_orm", b="https://embed.typeform.com/"; if(!gi.call(d,id)) { js=ce.call(d,"script"); js.id=id; js.src=b+"embed.js"; q=gt.call(d,"script")[0]; q.parentNode.insertBefore(js,q) } })() </script> <div style="font-family: Sans-Serif;font-size: 12px;color: #999;opacity: 0.5; padding-top: 5px;"> powered by <a href="https://admin.typeform.com/signup?utm_campaign=ktD2rd&utm_source=typeform.com-01DH8PS45JWZKYBJ6B37YX56NZ-pro&utm_medium=typeform&utm_content=typeform-embedded-poweredbytypeform&utm_term=EN" style="color: #999" target="_blank">Typeform</a> </div>
+
+In the meantime, [create a team](https://codesandbox.io/dashboard/teams/new) and invite your friends and co-workers to try out the all-new CodeSandbox Live today.
```
Comment (MichaelDeBoey) [inline packages/homepage/content/articles/redefining-code-collaboration/index.md]:
```suggestion
In the meantime, [create a team](/dashboard/teams/new) and invite your friends and co-workers to try out the all-new CodeSandbox Live today.
```

**Comment by MichaelDeBoey** [comment]
@CompuIves The image is deleted in #3755

**Pull request merged**